swf/src/shapes.rs

```diff
@@ -93,12 +93,12 @@ impl Shape {
     }
 
     #[inline]
-    fn first(&self) -> LineSegment {
+    fn first(&self) -> &LineSegment {
```

Comment:
Why this change? `LineSegment` fits in a single SIMD register; there's little point in returning references.

Reply:
There was an error and I simply applied the suggested fix. No other reason.

Reply:
Dereferenced it now instead.